src/components/uielements/SmallTile.astro

```diff
@@ -31,7 +28,7 @@ const HeadlineTagName = headlineTag;
       set:html={headline}
     />
     <span class="text-center text-sm font-light">
-      <PortableText value={description} />
+      <PortableText value={description} components={defaultComponents} />
```

Comment:
This components portabletext had custom styling.

Reply:
Two reasons: 
- I think text and headings should remain consistent throughout the website. Makes no sense to have a slightly thicker title for blog posts which is hardly noticable Maybe ask a frontender about their opinion as well.. @CosmicEvil @maartjegroenen 

H3 Before:
`<h3 class="text-balance font-serif text-[1.375rem] font-medium sm:text-lg">`

H3 After:
`<h3 class="text-balance font-serif text-xl font-medium sm:text-2xl">`

Even better would be to have a central styling for heading and span tags

- Second reason is that the custom youtube, image and link components don't work with the old portable text settings

Reply:
You should discuss this with Manon Wouters. She made the design.

Reply:
And it doesn't make sense to have all the headings etc styled the same. Sometimes you want to use a h2 where it is semantically correct to use a h2 but you don't want it to be as huge as a title h2 for example.

Reply:
fyi two portable texts have changed from custom to default styling.

1. Blog post intro (which is very different and needs to be reviewed) 
Currently:
![image](https://github.com/user-attachments/assets/bbbb50dc-ab4f-4200-b35e-e81160a7564b)
On dreamy trifle design:
![image](https://github.com/user-attachments/assets/d6ace802-e006-4018-a749-4b98080ca187)

2. highlightblock text, which is only different for heading tags which you probably won't use anyway in the regular text.
Currently:
![image](https://github.com/user-attachments/assets/42beaca2-f311-4450-a121-e468cfde5360)

Design:
![image](https://github.com/user-attachments/assets/c69fc7c6-851c-4707-9ee4-507ba583b9ff)


so all in all there's just nothing that actually changed

Reply:
But the heading tags are now not correct no? It used to be that the heading tags on the cms version were the same as dreamy trifle.

Reply:
these changes don't affect the cms version. Not sure if I understand what you mean? Which heading tags were affected by this?

Reply:
Ok then this PR can be merged